commcare-cloud/commcare_cloud/commands/ansible/ansible_playbook.py

```diff
@@ -229,3 +235,84 @@ def run(self, args, unknown_args):
         args.playbook = 'deploy_stack.yml'
         unknown_args += ('--tags=users',)
         AnsiblePlaybook(self.parser).run(args, unknown_args)
+
+
+class ManageServices(AnsibleOptions, _AnsiblePlaybookAlias):
+    """
+    example usages
+    1. To restart riak and stanchion only for riakcs
+       only option can be skipped to restart all services which are a part of riakcs
+       i.e riak, riak-cs and stanchion in that order
+        commcare-cloud staging service riakcs restart --only riak stanchion riak-cs
+    2. To start services under proxy i.e nginx
+       You can still pass ansible options which are
+       --user, --become, --become_user
+        commcare-cloud staging service proxy start -b
+    """
+    command = 'service'
+    help = (
+        "Manage services."
+    )
+    SERVICES = {
+        'proxy': 'nginx',
+        'riakcs': ['riak', 'riak-cs', 'stanchion']
+    }
+    ACTIONS = ['start', 'stop', 'restart']
+
+    def make_parser(self):
+        super(ManageServices, self).make_parser()
+        self.parser.add_argument(
+            'inventory_group',
+            choices=self.SERVICES,
+            help="The inventory group to run the command on"
+            )
+        self.add_ansible_options()
+        self.parser.add_argument(
+            'action',
+            choices=self.ACTIONS,
+            help="What action to take"
+        )
+        self.parser.add_argument(
+            '--only',
+            nargs='+',
+            help=(
+                "Specific services to act on for the inventory group. "
+                "Can mention multiple separated with empty spaces. "
+                "Example Usage: --only riak stanchion"
+            )
+        )
+
+    def for_proxy(self, args, unknown_args):
+        action = args.action
+        if action == 'start':
+            state = 'started'
+        elif action == 'restart':
+            state = 'restarted'
+        elif action == 'stop':
+            state = 'stopped'
+        args.module = 'service'
+        args.module_args = "name=nginx state=%s" % state
+        RunAnsibleModule(self.parser).run(
+            args,
+            unknown_args
+        )
+
+    def for_riakcs(self, args, unknown_args):
+        tags = []
+        action = args.action
+        args.playbook = "restart_riakcs.yml"
```

Comment:
I'd still push for these all to be in a directory together (`ansible/service_playbooks/restart_riakcs.yml` or something).